chapter1/rotateMatrix.js

```diff
@@ -0,0 +1,44 @@
+// Given an image represented by an NxN matrix, where each pixel in the image is 4 bytes, write a method to rotate the image by 90 degrees.
+
+/*
+Input:
+[
+  [1,2,3],
+  [4,5,6],
+  [7,8,9]
+];
+Output: 
+[
+  [7,4,1],
+  [8,5,2],
+  [9,6,3]
+];
+
+*/
+rotateMatrix = (arr) => {
```

Comment:
1. Functions defined in this format should have a var declaration like `const rotateMatrix = (arr) => {`
2. An arrow function isn't necessary here because you don't need execute it in the same scope as the parent scope. Just define as `function rotateMatrix(arr) {`